autospotting.go

```diff
@@ -55,21 +56,28 @@ func (c *cfgData) initialize() {
 	c.parseCommandLineFlags()
 	c.BuildNumber = string(build)
 
+	log.Printf("Current Configuration: %+v\n", c)
+
 	err := c.RawInstanceData.LoadFromAssetContent(instanceInfo)
 	if err != nil {
 		log.Fatal(err.Error())
 	}
 }
 
 func (c *cfgData) parseCommandLineFlags() {
-	flag.StringVar(&c.Regions, "regions", "", "Regions (comma separated list)"+
-		"where it should run, by default runs on all regions")
-	flag.Int64Var(&c.MinOnDemandNumber, "minOnDemandNumber", 0, "Minimum "+
-		"on-demand instances (number) running in ASG.")
-	flag.Float64Var(&c.MinOnDemandPercentage, "minOnDemandPercentage", 0.0,
-		"Minimum on-demand instances (percentage) running in ASG.")
+
+	flag.StringVar(&c.Regions, "regions", "",
+		"Regions where it should be activated (comma or whitespace separated list), by default it runs on all regions.")
+
+	flag.Int64Var(&c.MinOnDemandNumber, "min_on_demand_number", 0,
+		"On-demand capacity (as absolute number) ensured to be running in each of your groups. Can be overridden on a per-group basis.")
+
+	flag.Float64Var(&c.MinOnDemandPercentage, "min_on_demand_percentage", 0.0,
+		"On-demand capacity (as percentage of total capacity) ensured to be running in each of your groups. Can be overridden on a per-group basis.")
```

Comment:
I think the term `total capacity` can be miss leading, as the percentage is based on the currently running instances.